### PR TITLE
Enclose true, false and nil in backquote on document comment

### PR DESCRIPTION
**Pull request by MakeNowJust**
To improve document ;)

### CODE REVIEW

File: src/string.cr [hunk 1]
```diff
@@ -270,10 +270,10 @@ class String
   # or if the resulting integer doesn't fit an Int32, an ArgumentError is raised.
   #
   # Options:
-  # * **whitespace**: if true, leading and trailing whitespaces are allowed
-  # * **underscore**: if true, underscores in numbers are allowed
-  # * **prefix**: if true, the prefixes "0x", "0" and "0b" override the base
-  # * **strict**: if true, extraneous characters past the end of the number are disallowed
+  # * **whitespace**: if `true`, leading and trailing whitespaces are allowed
+  # * **underscore**: if `true`, underscores in numbers are allowed
+  # * **prefix**: if `true`, the prefixes `"0x"`, `"0"` and `"0b"` override the base
```
Comment (mverzilli) [inline src/string.cr]: Wouldn't it be better to remove the double quotes so that: `0x`, `0` and `0b`

Comment (MakeNowJust) [inline src/string.cr]: I think they mean string literal, does not?

Comment (mverzilli) [inline src/string.cr]: You're absolutely right, thanks for clarifying :).

**Comment by MakeNowJust** [comment]
@mverzilli Thanks!

**Pull request merged**